lrauv_system_tests/worlds/acoustic_comms_multi_vehicle.sdf

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" ?>
+
+<sdf version="1.6">
+  <world name="multi_lrauv">
+    <scene>
+      <!-- For turquoise ambient to match particle effect -->
+      <ambient>0.0 1.0 1.0</ambient>
+      <background>0.0 0.7 0.8</background>
+    </scene>
+
+    <physics name="1ms" type="ode">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="gz-sim-physics-system"
+      name="gz::sim::systems::Physics">
+    </plugin>
+    <plugin
+      filename="gz-sim-user-commands-system"
+      name="gz::sim::systems::UserCommands">
+    </plugin>
+    <plugin
+      filename="gz-sim-scene-broadcaster-system"
+      name="gz::sim::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="gz-sim-contact-system"
+      name="gz::sim::systems::Contact">
+    </plugin>
+
+    <plugin
+      filename="gz-sim-buoyancy-system"
+      name="gz::sim::systems::Buoyancy">
+      <uniform_fluid_density>1000</uniform_fluid_density>
+    </plugin>
+
+    <plugin
+      filename="gz-sim-particle-emitter-system"
+      name="gz::sim::systems::ParticleEmitter">
+    </plugin>
+
+    <plugin
+      filename="gz-sim-acoustic-comms-system"
+      name="gz::sim::systems::AcousticComms">
+      <max_range>2500</max_range>
+      <speed_of_sound>1000</speed_of_sound>
+    </plugin>
+
+    <light type="directional" name="sun">
+      <cast_shadows>true</cast_shadows>
+      <pose>0 0 10 0 0 0</pose>
+      <diffuse>1 1 1 1</diffuse>
+      <specular>0.5 0.5 0.5 1</specular>
+      <attenuation>
+        <range>1000</range>
+        <constant>0.9</constant>
+        <linear>0.01</linear>
+        <quadratic>0.001</quadratic>
+      </attenuation>
+      <direction>-0.5 0.1 -0.9</direction>
+    </light>
+
+    <include>
+      <pose>-5 0 0 0 0 0</pose>
+      <uri>https://fuel.ignitionrobotics.org/1.0/mabelzhang/models/Turquoise turbidity generator</uri>
+    </include>
+
+    <include>
+      <pose>0 0 1 0 0 1.57</pose>
```

Comment:
@adityapande-1995 hmm, do we have to reproduce Tethys' descriptions here?

Reply:
I tried changing the file ``acostic_comms_multi_vehicle.sdf`` like this :
```
<!--
  Development of this module has been funded by the Monterey Bay Aquarium
  Research Institute (MBARI) and the David and Lucile Packard Foundation
-->
<sdf version="1.9">
  <world name="multi_lrauv">
    <scene>
      <!-- For turquoise ambient to match particle effect -->
      <ambient>0.0 1.0 1.0</ambient>
      <!-- For default gray ambient -->
      <!--background>0.8 0.8 0.8</background-->
      <background>0.0 0.7 0.8</background>
    </scene>

    <physics name="1ms" type="ode">
      <max_step_size>0.001</max_step_size>
      <real_time_factor>1.0</real_time_factor>
    </physics>
    <plugin
      filename="gz-sim-physics-system"
      name="gz::sim::systems::Physics">
    </plugin>
    <plugin
      filename="gz-sim-user-commands-system"
      name="gz::sim::systems::UserCommands">
    </plugin>
    <plugin
      filename="gz-sim-scene-broadcaster-system"
      name="gz::sim::systems::SceneBroadcaster">
    </plugin>

    <plugin
      filename="gz-sim-imu-system"
      name="gz::sim::systems::Imu">
    </plugin>
    <plugin
      filename="gz-sim-magnetometer-system"
      name="gz::sim::systems::Magnetometer">
    </plugin>
    <plugin
      filename="gz-sim-buoyancy-system"
      name="gz::sim::systems::Buoyancy">
      <uniform_fluid_density>1025</uniform_fluid_density>
    </plugin>
    <plugin
      filename="gz-sim-acoustic-comms-system"
      name="gz::sim::systems::AcousticComms">
      <max_range>2500</max_range>
      <speed_of_sound>1000</speed_of_sound>
    </plugin>

    <plugin
      filename="gz-sim-particle-emitter2-system"
      name="gz::sim::systems::ParticleEmitter2">
    </plugin>

    <light type="directional" name="sun">
      <cast_shadows>true</cast_shadows>
      <pose>0 0 10 0 0 0</pose>
      <diffuse>1 1 1 1</diffuse>
      <specular>0.5 0.5 0.5 1</specular>
      <attenuation>
        <range>1000</range>
        <constant>0.9</constant>
        <linear>0.01</linear>
        <quadratic>0.001</quadratic>
      </attenuation>
      <direction>-0.5 0.1 -0.9</direction>
    </light>

    <include>
      <pose>-5 0 0 0 0 0</pose>
      <uri>turbidity_generator</uri>
    </include>

    <include>
      <pose>-15 0 1 0 0 0</pose>
      <uri>tethys_equipped</uri>
      <name>triton</name>

      <experimental:params>
        <plugin element_id="gz::sim::systems::CommsEndpoint" action="modify">
          <address>1</address>
          <topic>1/rx</topic>
        </plugin>
      </experimental:params>
    </include>

    <include>
      <pose>0 0 1 0 0 0</pose>
      <uri>tethys_equipped</uri>
      <name>tethys</name>

      <experimental:params>
        <plugin element_id="gz::sim::systems::CommsEndpoint" action="modify">
          <address>2</address>
          <topic>2/rx</topic>
        </plugin>
      </experimental:params>
    </include>

    <include>
      <pose>-15 0 1 0 0 0</pose>
      <uri>tethys_equipped</uri>
      <name>daphne</name>

      <experimental:params>
        <plugin element_id="gz::sim::systems::CommsEndpoint" action="modify">
          <address>3</address>
          <topic>3/rx</topic>
        </plugin>
      </experimental:params>
    </include>

  </world>
</sdf>

```

but the test is throwing this weird error : 

```
[Dbg] [JointPositionController.cc:215] initial_position: [0]
[Dbg] [SystemManager.cc:58] Loaded system [gz::sim::systems::JointPositionController] for entity [7]
[Wrn] [SystemLoader.cc:54] Trying to load deprecated plugin [ignition-gazebo-lift-drag-system]. Using [gz-sim-lift-drag-system] instead.
[Dbg] [SystemManager.cc:58] Loaded system [gz::sim::systems::LiftDrag] for entity [7]
[Wrn] [SystemLoader.cc:54] Trying to load deprecated plugin [ignition-gazebo-lift-drag-system]. Using [gz-sim-lift-drag-system] instead.
[Dbg] [SystemManager.cc:58] Loaded system [gz::sim::systems::LiftDrag] for entity [7]
[Wrn] [SystemPaths.cc:162] Finding plugins using deprecated IGN_ prefixed environment variable [IGN_GAZEBO_SYSTEM_PLUGIN_PATH]. Please use the GZ_ prefix instead.
[Wrn] [SystemLoader.cc:94] Found plugin [TethysCommPlugin] using deprecated environment variable [IGN_GAZEBO_SYSTEM_PLUGIN_PATH]. Please use [GZ_SIM_SYSTEM_PLUGIN_PATH] instead.
[libprotobuf ERROR google/protobuf/descriptor_database.cc:58] File already exists in database: lrauv_ignition_plugins/lrauv_state.proto
[libprotobuf FATAL google/protobuf/descriptor.cc:1358] CHECK failed: GeneratedDatabase()->Add(encoded_file_descriptor, size): 
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():  CHECK failed: GeneratedDatabase()->Add(encoded_file_descriptor, size): 
[1]    67877 abort (core dumped)  ./build/lrauv_system_tests/test/vehicle/test_acoustic_comms
```

Reply:
You can have only one test per file unfortunately.
See: https://github.com/osrf/lrauv/issues/159

We have not figured out why this happens. I don't think we should be copying the model descriptions in.

Reply:
Done https://github.com/osrf/lrauv/pull/243/commits/1aeb14732c83d553b533dcc38e1c846aa3369609